tasks/zadanie_5.php

```diff
@@ -22,7 +22,7 @@
 $givenMoneyByClient = 10000;
 $costOfProducts = 8515;
 
-$rest = 0; // TODO
-$restPennies = 0; // TODO
+$rest = floor(($givenMoneyByClient - $costOfProduc)/100); // TODO
```

Comment:
Tutaj i poniżej literki trochę zjadło :D 
```suggestion
$rest = floor(($givenMoneyByClient - $costOfProducts)/100); // TODO
```